<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Code Analyser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="MAIN_SPLITTER">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="FILES_BROWSERS_TABS">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="WELCOME">
        <attribute name="title">
         <string>Welcome</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_6"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="DUMMY">
        <attribute name="title">
         <string>Dummy</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_7"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="LOGGERS_LAYOUT" stretch="1,0">
        <item>
         <widget class="QSplitter" name="SPLITTER_LOGGERS">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTabWidget" name="TOOLS_LOGGERS_TAB">
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="TOOL_PYLINT_LOGGER">
            <attribute name="icon">
             <iconset>
              <normaloff>../images/ruby.png</normaloff>../images/ruby.png</iconset>
            </attribute>
            <attribute name="title">
             <string>PyLint</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="GenericToolLoggerView" name="TOOL_PYLINT_LOGGER_VIEW"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="TOOL_PYFLAKES_LOGGER">
            <attribute name="icon">
             <iconset>
              <normaloff>../images/sport_basketball.png</normaloff>../images/sport_basketball.png</iconset>
            </attribute>
            <attribute name="title">
             <string>PyFlakes</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="GenericToolLoggerView" name="TOOL_PYFLAKES_LOGGER_VIEW"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="TOOL_PYMETRICS_LOGGER">
            <attribute name="icon">
             <iconset>
              <normaloff>../images/rainbow.png</normaloff>../images/rainbow.png</iconset>
            </attribute>
            <attribute name="title">
             <string>PyMetrics</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="GenericToolLoggerView" name="TOOL_PYMETRICS_LOGGER_VIEW"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="GenericToolLoggerView" name="PROCESSES_LOGGER"/>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="BTNS_LAYOUT">
          <item>
           <widget class="QToolButton" name="BTN_CLEAR_LOGGERS">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../pyanalysers/images/Clear32x32.png</normaloff>
              <normalon>../images/Clear32x32.png</normalon>../pyanalysers/images/Clear32x32.png</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="BTN_PAUSE_LOGGERS">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../images/Pause32x32.png</normaloff>
              <normalon>../images/Play32x32.png</normalon>../images/Pause32x32.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="BTN_SAVE_LOGGERS_AS_TEXT">
            <property name="toolTip">
             <string>Save to Text...</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../images/save.png</normaloff>../images/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="BTN_SAVE_LOGGERS_AS_HTML">
            <property name="toolTip">
             <string>Save to Html...</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../images/save.png</normaloff>../images/save.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MENUBAR">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="PYLINT_ACTION"/>
    <addaction name="PYFLAKES_ACTION"/>
    <addaction name="separator"/>
    <addaction name="PYMETRICS_ACTION"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ABOUT_ACTION"/>
    <addaction name="HELP_ACTION"/>
   </widget>
   <widget class="QMenu" name="actionDialogs">
    <property name="title">
     <string>Dialogs</string>
    </property>
    <addaction name="FILEINFOSET_EDITOR"/>
    <addaction name="TOOLS_CONFIGURATIONS"/>
    <addaction name="TOOLS_OUTPUT_CONFIGURATIONS"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionsMenu"/>
   <addaction name="actionDialogs"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="TOOLBAR">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="PYLINT_ACTION"/>
   <addaction name="PYFLAKES_ACTION"/>
   <addaction name="PYMETRICS_ACTION"/>
   <addaction name="separator"/>
   <addaction name="DUMMY_CHOICE_FILES_DATASET"/>
   <addaction name="separator"/>
   <addaction name="RUN_TOOLS_ACTION"/>
   <addaction name="STOP_TOOLS_ACTION"/>
   <addaction name="separator"/>
   <addaction name="DUMMY_CHOICE_NB_PROCESSES"/>
   <addaction name="separator"/>
   <addaction name="FILEINFOSET_EDITOR"/>
   <addaction name="TOOLS_CONFIGURATIONS"/>
   <addaction name="TOOLS_OUTPUT_CONFIGURATIONS"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="PYLINT_ACTION">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/ruby.png</normaloff>../images/ruby.png</iconset>
   </property>
   <property name="text">
    <string>Run PyLint</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="PYFLAKES_ACTION">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/sport_basketball.png</normaloff>../images/sport_basketball.png</iconset>
   </property>
   <property name="text">
    <string>Run PyFlakes</string>
   </property>
   <property name="iconText">
    <string>Run PyFlakes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="PYCHECKER_ACTION">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/sport_soccer.png</normaloff>../images/sport_soccer.png</iconset>
   </property>
   <property name="text">
    <string>Run PyChecker</string>
   </property>
   <property name="toolTip">
    <string>Run PyChecker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, C</string>
   </property>
  </action>
  <action name="PEP8_ACTION">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/golf_ball.png</normaloff>../images/golf_ball.png</iconset>
   </property>
   <property name="text">
    <string>Run Pep8</string>
   </property>
   <property name="toolTip">
    <string>Run Pep8</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, C</string>
   </property>
  </action>
  <action name="PYMETRICS_ACTION">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/rainbow.png</normaloff>../images/rainbow.png</iconset>
   </property>
   <property name="text">
    <string>Run PyMetrics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="DUMMY_CHOICE_FILES_DATASET">
   <property name="text">
    <string>DataSet</string>
   </property>
   <property name="toolTip">
    <string>Select Files DataSet</string>
   </property>
  </action>
  <action name="RUN_TOOLS_ACTION">
   <property name="icon">
    <iconset>
     <normaloff>../images/run.png</normaloff>../images/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Selected Tools</string>
   </property>
   <property name="toolTip">
    <string>Run selected tools</string>
   </property>
  </action>
  <action name="STOP_TOOLS_ACTION">
   <property name="icon">
    <iconset>
     <normaloff>../images/cross.png</normaloff>../images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Stop Selected Tools</string>
   </property>
   <property name="toolTip">
    <string>Stop running tools</string>
   </property>
  </action>
  <action name="TOOLS_CONFIGURATIONS">
   <property name="icon">
    <iconset>
     <normaloff>../images/Wrench.png</normaloff>../images/Wrench.png</iconset>
   </property>
   <property name="text">
    <string>Open Tools Configurations...</string>
   </property>
  </action>
  <action name="TOOLS_OUTPUT_CONFIGURATIONS">
   <property name="icon">
    <iconset>
     <normaloff>../images/Repair.png</normaloff>../images/Repair.png</iconset>
   </property>
   <property name="text">
    <string>Open Tools Output Configurations...</string>
   </property>
  </action>
  <action name="FILEINFOSET_EDITOR">
   <property name="icon">
    <iconset>
     <normaloff>../images/FilesConfig.png</normaloff>../images/FilesConfig.png</iconset>
   </property>
   <property name="text">
    <string>Files DataSet Editor...</string>
   </property>
  </action>
  <action name="DUMMY_CHOICE_NB_PROCESSES">
   <property name="text">
    <string>Nb. Processes</string>
   </property>
   <property name="toolTip">
    <string>Select Nomber of Processes</string>
   </property>
  </action>
  <action name="ABOUT_ACTION">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="HELP_ACTION">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericToolLoggerView</class>
   <extends>QTextEdit</extends>
   <header>generictoolloggerview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editabletreemodel.qrc"/>
 </resources>
 <connections/>
</ui>
